Examples/ObjectiveC/OrnamentsLayoutExample.m

```diff
@@ -0,0 +1,41 @@
+#import "OrnamentsLayoutExample.h"
+
+@import Mapbox;
+
+// Ornament positions matrix arry, this example will demonstrate how ornament looks like in different positions.
```

Comment:
Does this comment still apply?

Reply:
Ah good catch, this one slipped past me. :)